project_event/report/project_task_activity_report.py

```diff
@@ -93,8 +93,8 @@ def get_different_department_and_employee(sorted_list):
         done = set()
         result = []
         for element in sorted_list:
-            if (element['employee'], element['department']) not in done:
-                done.add((element['employee'], element['department']))
+            if element['employee'] not in done:
```

Comment:
I thought the set (employee, department) was a specification. This spec changed multiple times.
So, even if an employee has two different departments we only display one?